integration_tests/test_store_history.py

```diff
@@ -25,53 +25,53 @@
 import integration_tests
 
 
-class HistoryTestCase(integration_tests.StoreTestCase):
+class RevisionsTestCase(integration_tests.StoreTestCase):
```

Comment:
That seems fine, although I don't see that here. Have you not pushed it? Make sure you add a test case for history as well, verifying that it's deprecated etc.